blablas/ep50/index.md

```diff
@@ -0,0 +1,81 @@
+---
+date: 2020-09-07
+time: 22h:15min
+duration: "01:06:16"
+title: "MSS With Mehdi Mourabit"
+tags: ["entrepreneurship", "startup", "digital marketing"]
+isNext: false
```

Comment:
you need to add `category: "MSS"` 
check here => https://github.com/DevC-Casa/geeksblabla.com#add-new-episode-